arviz/__init__.py

```diff
@@ -1,6 +1,6 @@
 # pylint: disable=wildcard-import,invalid-name,wrong-import-position
 """ArviZ is a library for exploratory analysis of Bayesian models."""
-__version__ = "0.8.2"
+__version__ = "0.8.3"
```

Comment:
Is it possible to run pymc3 ci (or tests locally in docker or something) using ArviZ master? If this were possible, I'd wait to bump the version, merge, run tests and once they pass bump version and release.

Reply:
I can put arviz@master into the requirements-dev. But the PyMC3 CI is currently far away from a text-matrix.. We'd risk to break compatibility with the releases and I don't know how to feel about that